invoicing_app/management/commands/generate_tax_receipts_new.py

```diff
@@ -93,19 +78,59 @@ def __init__(self, *args, **kwargs):
         self.event_id = None
         self.user_id = None
         self.sentry = logging.getLogger('sentry')
-
+        self.query = '''
+                    SELECT
+                        `Orders`.`event` AS `event_id`,
+                        `Events`.`uid` AS `event__user_id`,
+                        `Payment_Options`.`epp_country` AS `event___paymentoptions__epp_country`,
+                        `Events`.`currency` AS `event__currency`,
+                        `Payment_Options`.`epp_name_on_account` AS `event___paymentoptions__epp_name_on_account`,
+                        `Payment_Options`.`epp_address1` AS `event___paymentoptions__epp_address1`,
+                        `Payment_Options`.`epp_address2` AS `event___paymentoptions__epp_address2`,
+                        `Payment_Options`.`epp_zip` AS `event___paymentoptions__epp_zip`,
+                        `Payment_Options`.`epp_city` AS `event___paymentoptions__epp_city`,
+                        `Payment_Options`.`epp_state` AS `event___paymentoptions__epp_state`,
+                        `Events`.`event_parent` AS `event__event_parent`,
+                        COUNT(`Orders`.`event`) AS `payment_transactions_count`,
+                        SUM(`Orders`.`eb_tax`) AS `total_tax_amount`,
+                        SUM(`Orders`.`mg_fee`) AS `total_taxable_amount_with_tax_amount`,
+                        SUM(`Orders`.`gross`) AS `base_amount`
+                    FROM `Orders`
+                        INNER JOIN `Events` ON (`Orders`.`event` = `Events`.`id` )
+                        INNER JOIN `Payment_Options` ON PARENT_CHILD_MASK
+                    WHERE (
+                        `Orders`.`status` = %(status_query)s AND
+                        `Orders`.`pp_date` <= %(localize_end_date_query)s AND
+                        `Orders`.`changed` >= %(localize_start_date_query)s AND
+                        `Orders`.`mg_fee` > '0.00' AND
+                        `Orders`.`changed` <= %(localize_end_date_query)s AND
+                        `Orders`.`pp_date` >= %(localize_start_date_query)s AND
+                        `Payment_Options`.`accept_eventbrite` = 1 AND
+                        `Payment_Options`.`epp_country` = %(declarable_tax_receipt_countries_query)s
+                        CONDITION_MASK
+                    )
+                    GROUP BY
+                        `event_id`
+                    ORDER BY NULL
+                '''
         super(Command, self).__init__(*args, **kwargs)
 
-    def localize_date(self, country_code, date):
```

Comment:
i think we dont need to remove this. Is useful for test the new script and the old script with the same timezone

Reply:
I didn't remove this.  I just move down 'cause improve the legibility of the code!